File: VB/DXApplication1/Root.Master.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Web.ASPxClasses

Namespace DXApplication1
	Partial Public Class Root
		Inherits System.Web.UI.MasterPage
		Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
			MainSplitter.GetPaneByName("HeaderPane").Size = If(ASPxWebControl.GlobalTheme = "Moderno", 101, 87)
		End Sub
	End Class
End Namespace

File: CS/DXApplication1/Root.Master.cs
using System;
using DevExpress.Web.ASPxClasses;

namespace DXApplication1 {
    public partial class Root : System.Web.UI.MasterPage {
        protected void Page_Load(object sender, EventArgs e) {
            MainSplitter.GetPaneByName("HeaderPane").Size = ASPxWebControl.GlobalTheme == "Moderno" ? 101 : 87;
        }
    }
}